serveur_calcul_python/tests/data_gen/calcs_input_data_store.py

```
@@ -0,0 +1,87 @@
+from classes import parking_inventory_inputs as PCI
+import pandas as pd
+def generate_all_required_inputs():
+    simple_inputs= generate_inputs_simple()
+    thresh_data = generate_thresh_data()
+    add_data = generate_addition_based_data()
+    floor_data = generate_floor_data()
+    ceil_data_or = generate_ceil_data_or()
+    ceil_data_thresh = generate_ceil_data_thresh()
+    output = pd.concat([simple_inputs,thresh_data,add_data,floor_data,ceil_data_or,ceil_data_thresh]).reset_index(drop=True)
+    return output
```

Ruff (0.15.12)

[warning] Unnecessary assignment to `output` before `return` statement

Remove unnecessary assignment

(RET504)